src/routes/plans/[id]/+page.svelte

```diff
@@ -346,9 +347,13 @@
     simulationDataAbortController?.abort();
     simulationDataAbortController = new AbortController();
     effects.getSpans(datasetId, data.user, simulationDataAbortController.signal).then(newSpans => ($spans = newSpans));
+    effects
+      .getEvents(datasetId, data.user, simulationDataAbortController.signal)
+      .then(newEvents => ($simulationEvents = newEvents));
   } else {
     simulationDataAbortController?.abort();
     $spans = [];
+    $simulationEvents = [];
```

Comment:
Could you clear this store in the `resetSimulationStores` function in `simulation.ts`?

Reply:
Done